dependencies.json

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
It would be interesting if this file could be replaced by: https://github.com/vert-x3/vertx-stack/tree/master/stack-manager/src/main/descriptor

This would ensure that the dependencies would be always up to date.